src/components/Search/SearchAutocomplete.tsx

```diff
@@ -48,6 +54,14 @@ const SearchAutocomplete: React.FC<{
 
   const zipCodeInput = checkForZipcodeMatch ? input : '';
 
+  useEffect(() => {
```

Comment:
How would you feel about making this a hook that either lived in this file or another one (maybe zipCodeUtils.tx or somethign) instead of putting all of the logic in the Search Autocomplete component. 

My thought would be a function like `useCountyZipCodeMap()` or something that did the fetch and returns a zipcode object